src/protocol/DocumentServer.js

```diff
@@ -222,6 +223,33 @@ define('polymer-designer/protocol/DocumentServer', [
       });
     }
 
+    backspaceText(request) {
+      let text = request.message.text;
+      let node = document.createTextNode(text);
```

Comment:
That's so very weird. I definitely want to understand what's going on there, so when you can get this rebased, let me know and I'll debug locally.

And of course, the Road to Polymer is a little more important to the community :D :+1: